src/index.html

```diff
@@ -2,16 +2,170 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action=""
+      autocomplete="off"
+      onsubmit="onSubmit()">
```

Comment:
```suggestion
    <form
      action="#"
      autocomplete="off"
      onsubmit="onSubmit()"
    >
```
Располагай закрывающую скобку на одном уровне с открывающей и не оставляй атрибут action пустым